File: transaction/js/MakerEventInfo.js
var request = require("request");
// set parameters
const maker = "0xfc898b18a70ce49579f8d79a32e29928c15b4bc8";
// set url
const url = "https://api.deversifi.com/v1/pub/events/maker/" + maker;
// execute request and print result
request(url, function(error, response, body) {
    console.log(JSON.parse(body));
});

File: transaction/python/MakerEventInfo.py
import requests
import json

# set the url
url = "https://api.deversifi.com/v1/pub/events/maker/{maker}"

# set the parameters
url = url.format(
    maker="0xfc898b18a70ce49579f8d79a32e29928c15b4bc8")

# standard headers
headers = {
    "Accept": "application/json",
    "Content-Type": "application/json"
}

# execute the request and load into dictionary
response = json.loads(requests.request(
    "GET", url, headers=headers).text)

# print the response from dictionary
print(json.dumps(response, indent=2))
